enhancements/service-catalog/service-catalog-and-broker-removal.md

```diff
@@ -0,0 +1,211 @@
+1. Remove the Template Service Broker from the OpenShift 4.4 release payload (ART)
+1. Remove the Template Service Broker from productization (ART)
+1. Remove the Ansible Service Broker from productization (ART)
+1. Remove the Template Service Broker Operator from productization (ART)
+1. Remove the Ansible Service Broker Operator from productization (ART)
+1. Remove the Service Catalog from productization (ART)
+
+### Non-Goals
+
+N/A
+
+## Proposal
+
+### User Stories
+
+#### Story 1
+
+Implement a mechanism that ensures the Service Catalog and Service Brokers are
+not present after an upgrade to OpenShift 4.4.
+
+We could potentially block upgrades from happening until the admin removes the
+Service Catalog and the brokers. This would need a mechanism that blocks
+upgrades. There is an `Upgradeable: False` in the operator's status but not sure
+if that is checked by the CVO during upgrades.
+
+We could let the upgrade occur and change the behavior of the existing operators
+to simply remove the objects they own. If we go this route, we will need to keep
+the operators in the release payload.
+
+Another mechanism, similar to the one above is to use a [Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/) to do the deletion. The job would be defined in the
+manifest directory of the Service Catalog operators.
+
+#### Story 2
+
+Remove Service Catalog and the brokers from the release payload
+
+1. Remove the Service Catalog from the OpenShift 4.4 release payload (ART)
+1. Remove the Ansible Service Broker from the OpenShift 4.4 release payload (ART)
+1. Remove the Template Service Broker from the OpenShift 4.4 release payload (ART)
+
+#### Story 3
+
+Remove the Brokers from productization
+
+1. Remove the Template Service Broker from productization (ART)
+1. Remove the Ansible Service Broker from productization (ART)
+1. Remove the broker sections from the documentation (Docs)
+
+#### Story 4
+
+Remove the broker operators from productization
+
+1. Remove the Template Service Broker Operator from productization (ART)
+1. Remove the Ansible Service Broker Operator from productization (ART)
+1. Remove the operator instructions from the documentation (Docs)
+
+#### Story 5
+
+1. Remove the Service Catalog from productization (ART)
+1. Remove the installation instructions from the documentation (Docs)
+1. Ensure you how to disable service catalog instructions are correct (Docs/Eng)
+
+### Implementation Details/Notes/Constraints [optional]
+
+There are still a few unknowns for exactly how to accomplish the removal of
+the Service Catalog and Service Brokers. I don't think any of them are show
+stoppers, just might cause a few changes to this proposal depending on the
+approach taken.
+
+### Risks and Mitigations
+
+The biggest risk is if there needs to be additional work done to the CVO in
+order to support anything we would like to do described in this proposal.
```

Comment:
Any chance we can close the loop on the CVO questions this week to better understand these risks?

Reply:
Removed, CVO is not likely to have any major changes.